CHANGELOG.md

```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- 
+
+### Changed
+
+- Refactor how `AddAssignUnchecked` and `SubAssignUnchecked` are used in `Erc20`, `Erc721`, and `Erc721Consecutive`. #467
+- Refactor `Nonces`. #467
```

Comment:
IMHO if anything in API has changed than we should mention this change. If not, no need.